africanus/gridding/simple/tests/test_simple_gridder.py

```diff
@@ -78,7 +78,12 @@ def test_degridder_gridder():
 
 
 @pytest.mark.parametrize("plot", [False])
-def test_psf_subtraction(plot):
+@pytest.mark.parametrize("oversample", [
+    1,
+    pytest.param(2, marks=pytest.mark.xfail(reason="Unknown")),
+    pytest.param(3, marks=pytest.mark.xfail(reason="Unknown")),
+])
+def test_psf_subtraction(plot, oversample):
```

Comment:
@landmanbester This test succeeds with an oversampling of 1, but fails with 2 and 3.

Reply:
@landmanbester This is probably still the entry point for seeing the discrepancy. Succeeds with oversampling of 1, but fails with 2 and 3.